contracts/registrar/IZNSRootRegistrar.sol

```diff
@@ -12,11 +12,12 @@ import { PaymentConfig } from "../treasury/IZNSTreasury.sol";
 struct CoreRegisterArgs {
     bytes32 parentHash;
     bytes32 domainHash;
-    address registrant;
-    address domainAddress;
+    string label;
+    address domainOwner;
+    address tokenOwner;
     uint256 price;
     uint256 stakeFee;
-    string label;
```

Comment:
why do we move label? we get rid of any potential gain from struct memory packing by having it in between addresses